src/utils/SiriusAdapter.cpp

```diff
@@ -124,7 +124,14 @@ class TOPPSiriusAdapter :
 
   void registerOptionsAndFlags_() override
   {
-    registerInputFile_("executable", "<executable>", "", "sirius executable e.g. sirius", false, false, ListUtils::create<String>("skipexists"));
+    registerInputFile_("executable", "<executable>", 
+      // choose the default value according to the platform where it will be executed
+#ifdef OPENMS_WINDOWSPLATFORM
```

Comment:
@jpfeuffer is this problematic if a workflow created on Linux is used on windows? and vice versa

Reply:
cant we just check for both?

Reply:
we could but this requires some extra coding... not worth the effort IMHO
ready for merge?